EIPS/eip-5450.md

```diff
@@ -162,7 +162,7 @@ It poses no risk to backwards compatibility, as it is introduced only for EOF1 c
 
 ## Security Considerations
 
-Needs discussion.
+As mentioned above, the proposed validation algorithm has linear computational and space complexity and its cost is covered by the [EIP-3860](./eip-3860.md) for initcode cost and high per-byte cost of deploying code.
```

Comment:
This is not correct. 3860 costs don't apply here, but transaction data costs do.

Reply:
done